src/en/developers/google-summer-of-code/index.md

```diff
@@ -66,21 +67,23 @@ A reduction in build times from Ceph developers pushing to the ceph-ci repo to r
 
 **Subcomponent of Ceph:** RGW
 
-**Description of project:**
+**Description of project:** 
 
-Bucket notifications are important building block for many use cases. And persistent bucket notifications in particular, as they let the system overcome broker outages. However, since the persistent notifications are backed with a RADOS queue, they have a cost. Both in the extra load on the RADOS cluster, and with the inability to operate in environments where there is no RADOS backend. In this project, we would like to implement persistent bucket notifications in the RADOS Gateway using a Redis Queue. Combined with the "zipper" project we would be able to enjoy bucket notifications with backends like posix, dbstore, daos etc.
+Bucket notifications are important building block for many use cases. And persistent bucket notifications in particular, as they let the system overcome broker outages. However, since the persistent notifications are backed with a RADOS queue, they have a cost. Both in the extra load on the RADOS cluster, and with the inability to operate in environemnts where there is no RADOS backend. In this project, we would like to implement persistent bucket notifications in the RADOS Gateway using a Redis Queue. Combined with the "zipper" project we would be able to enjoy bucket notifications with backends like posix, dbstore, daos etc.
 
-**Expected Outcome(s):**
+**Standup/weekly call mentee could attend?:** RGW standup
 
-Persistent bucket notifications being stored in a Redis cluster that is being read and written to by the RADOS Gateway with non RADOS backends like posix store
+**Steps to evaluate an applicant for the project:**
+
+TBD, A link with more detailed instructions will be provided
 
 <hr class="hr">
 
 ## RGW AdminAPI Audit & Refactoring
 
-**Mentor name(s):** Ali Maredia, Samarah Uriarte
```

Comment:
@VallariAg the changes from here on down should be removed from this PR since they have nothing to do with the Shaman project. Maybe pull your branch up to main?

Reply:
@alimaredia that was my fault, rebased and pushed.